### DISCUSSION TITLE
FVM WASM Resources

### DISCUSSION THREAD

**Discussion by Gekctek** [filecoin-project/community#612] [Developers]
I am evaluating the feasibility of integrating Motoko into FVM and have a few questions. (Its hard to differentiate FVM news with FEVM news)
1) What is the best way, if at all, to run a wasm actor locally/on a test net. I know FEVM is the big news and WASM comes later, but curious on the roadmap for testing at least
2) Right now the docs for WASM import functions for sys calls that I am using is the Rust sdk sys module docs. Is that the best place or is there a documented list somewhere else
3) In general I am curious how much should I expect the FVM WASM spec to change, or is it somewhat settled?

**Comment by domwoe** [comment]
For everyone reading this and asking themselves "WTF is Motoko?"

[Motoko](https://github.com/dfinity/motoko) is a modern high-level programming language, designed by Andreas Rossberg (one of the WASM creators), to simplify the development of distributed and decentralized applications on the [Internet Computer](https://internetcomputer.org).

**Reply by lyjmry** [reply]
hello, nice to see you here
